crates/tabby/src/main.rs

```diff
@@ -54,12 +55,20 @@ pub enum Commands {
     /// Execute the repository sync job.
     #[cfg(feature = "ee")]
     #[clap(name = "job::sync")]
-    JobSync,
+    JobSync(JobArgs),
 
     /// Execute the index job.
     #[cfg(feature = "ee")]
     #[clap(name = "job::index")]
-    JobIndex,
+    JobIndex(JobArgs),
+}
+
+#[derive(clap::Args)]
+pub struct JobArgs {
+    #[clap(long, requires = "address")]
```

Comment:
what does `requires = "address"` mean here?

Reply:
If you supply the token, you must also supply the address.